components/learn/modules/ROOT/pages/dip-staking.adoc

```diff
@@ -2,86 +2,120 @@
 * Red dot: The locked capital in the risk bundle reached 100% or more of the total supported capital. +
 
 When hovering the mouse over the dots, you will see the stake usage in percent, the locked capital in USDT and the total supported capital in USDT.  +
 You can sort the table by any information. Just click on the information. Default is sorted by 'Active until.’ You can recognize this by the small arrow to the right of the text.
 
-*DETAILS* +
+==== DETAILS
 On the table's far right, you can click '`DETAILS.`'
 
+== Manage your DIP staking
+
+In the detail view, you can stake DIP token or manage your staked DIP token.
+
 image::_images/dip-3-details.png[]
 
 Here you will find detailed information about the risk bundle. The general information and personal information like '`My Supported Capital.`' +
 The buttons where you can act are highlighted in blue. +
-*STAKE* +
+
+==== STAKE
 You can start staking or stake more DIP token in this risk bundle.  +
-*UNSTAKE* +
+
+==== UNSTAKE
 The '`UNSTAKE`' button in the screenshot is not active. You cannot unstake your DIP token in the example screen because the risk bundle is in the state '`Active.`' Unstaking is only available for expired risk bundles or in state '`Closed`' or '`Burned.`' +
 Please note: Your DIP token are not at risk. +
-*CLAIM REWARDS* +
-You can claim your rewards at any time during the lifetime of a risk bundle, but at the latest when you unstake the total amount of your staked DIP tokens for this bundle.
 
-=== Stakes
+==== CLAIM REWARDS
+You can claim your rewards at any time during the lifetime of a risk bundle, but at the latest when you unstake the total amount of your staked DIP tokens for this bundle. +
+
+==== RESTAKE
+
+Restaking is only possible in risk bundles that are already closed or expired. You can only restake the total amount of DIP token you staked in this risk bundle. You can only restake in a risk bundle in which you have not yet staked any DIP token.
+
+== Tab '`Stakes`'
 
 This area is identical to the '`DIP Staking`' menu item. You can see all existing risk bundles.
 
-=== Stake DIP
-test
+== Tab '`Stake DIP`'
+
 image::_images/dip-4-stakingdip.png[]
 
 In this area, you will only see the risk bundles in which you can stake DIP token. By clicking '`SELECT`' on the right, compare the conditions and terms and choose a risk bundle. You see all details of the risk bundle.
-the minimum volume is 5,000 DIP
+The minimum volume is 5,000 DIP.
+
+==== Gasless staking
+
+By checking the box, '`I would like Etherisc to submit the transaction and pay fees on my behalf,`' you can set Etherisc to pay the fees. 
+
+IMPORTANT: Etherisc actually limits the fees to 35 Gwei. You can see the current limit by hovering over the info button. The text that appears contains the current limit. Etherisc does not guarantee that the transaction will be executed immediately.
```

Comment:
This was changed to 20. I would recommend to remove the exact number from the docs as it might change dynamically. The current value is shown in the webapp when hovering over the (i) button next to the text.